group_project/public/css/group_project.css

```diff
@@ -172,17 +172,22 @@
   padding: 10px 0;
   font-size: 14px;
 }
+.upload_form .upload_item .upload_input{
```

Comment:
CSS elements should be scoped to the XBlock div object - this is important to avoid applying the CSS at other places in the LMS where it wouldn't be expected to apply:

```

```

I know you are just following the pattern from the rest of this file, but it would be worth adding the proper scoping at least to the rules you're creating/modifying.